rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -2579,6 +2592,13 @@ void game_list_frame::PopulateGameList()
 			}
 		}
 
+		QString directory_size = tr("Unknown");
+
+		if (usz size = fs::get_dir_size(game->info.path); size != umax)
```

Comment:
this is bad. the size shouldn't be calculated in a GUI refresh. it should be calculated on a hard refresh